pkg/image/apis/image/helper.go

```diff
@@ -41,6 +46,16 @@ const (
 	ImportRegistryNotAllowed = "registry is not allowed for import"
 )
 
+// knownSignatureStores is a list of known signature stores. The key represents
+// a registry name and it maps to a signature store for that registry. The
+// signature store has to be HTTPS and it must offer image signatures based on
+// this format: https://STORE_URL/IMAGE_NAME/signature-1
+// The response must be valid image signature blob.
+// TODO: Maybe this should be configurable via master config?
```

Comment:
> We're presuming for this use case the administrator controls the location of the sigstore for a set of trusted registries. 

Is it possible for non-admin tenants to create their own image streams, choosing their own external registries? If so, they need to be able to point at the corresponding external sigstore servers, without admin help and without recompiling OpenShift.